docs/how-to/h-manage-app.md

```diff
@@ -16,27 +16,31 @@ To remove a relation:
 juju remove-relation kafka application
 ```
 
-## Outside juju or for charms not implementing `kafka_client`
+## Outside Juju or for charms not implementing `kafka_client`
 
-The `kafka_client` interface is used with the `data-integrator` charm. This charm allows to automatically create and manage product credentials needed to authenticate with different kinds of data platform charmed products:
+The `kafka_client` interface is used with the `data-integrator` charm. This charm can automatically create and manage product credentials needed to authenticate with different kinds of data platform Charmed products.
```

Comment:
Uhm, `Charmed Spark` is a name of the product, but I'm not sure `Charmed` can be seen as a name. I believe we are using `charmed` more as an adjective, not quite a substantive

Reply:
That's highly experimental, yes, but I'd like to suggest using "Charmed" as a brand. Charmed Kafka - both start with upper case, Charmed product - "Charmed" is capitalized to distinguish from generic meaning of the word, like an enchanted talisman. 
Let me know what you think.

Reply:
Uhm, I'm not sure I'm thrilled by this. But this is not my call. Maybe just talk to @a-velasco and get her thoughts on this as well, since it seems this would be something that we need to get consistent across charms (possibly even beyond Data Platform charms, but more in the ecosystem).

Reply:
@deusebio I personally think it makes sense because it's a proper adjective (derived from a proper noun), which means that in English it should technically be capitalized.

I definitely understand that it looks strange though. These types of proper adjectives aren't usually capitalized in Spanish, Portuguese (and I'm guessing Italian) so it takes some getting used to. I'm currently asking the Juju author for a final, extra qualified opinion 😄

Reply:
Welp - nevermind, seems the rule is that we never capitalize it (unless at the beginning of a sentence). Her point was that if we adhered to that rule and capitalized all the Juju jargon, sentences would look kind of ridiculous with all the caps.

Reply:
> Welp - nevermind, seems the rule is that we never capitalize it (unless at the beginning of a sentence). Her point was that if we adhered to that rule and capitalized all the Juju jargon, sentences would look kind of ridiculous with all the caps.

Well, it was worth a try. Thank you for clearing this out. Is it a written rule? I'm asking to know if there is a place where I can make a PR to suggest the other way of doing it. But for now, I'd stick to the rule, even if it's unwritten.

Reply:
I'm not sure too sure - it's definitely not somewhere I can easily find in the Juju docs. I'll message you more details privately so this convo can be resolved in the PR.

Reply:
Yes. I guess I actually never understood the rule for capitalizing Italian, other than I had to :D

Anyhow, thanks for looking into this! I'll resolve the conversation then!